teaching_platform/projects/db_handler.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+from teaching_platform.projects import exceptions
+from teaching_platform.models import User, Role, Project
+from teaching_platform.extensions import db
+
+def add_project(name, mentor_id, status="In development."):
+    mentor = User.query.filter_by(id=mentor_id, role=Role.TEACHER).first()
+    if not mentor: raise exceptions.MentorDosentExistError()
+    project = Project(mentor=mentor, name=name, status=status)
+
+    db.session.add(project)
+    db.session.commit()
+
+def get_all_projects():
+    return Project.query.all()
+
+def get_project(project_id):
+    project = Project.query.filter_by(id = project_id).first()
+    if not project: raise exceptions.ProjectDosentExistError()
+
+    return project
+
+def remove_project(project_id):
+    query = Project.query.filter_by(id=project_id)
```

Comment:
`project` instead of `query` as variable name just to be consistent.

Reply:
It's named `query `and not `project `because variable `project `nontains actual `Project ` object, and `query `is just query to be reused in deleting.